make/testing_flags.mk

```diff
@@ -35,7 +35,7 @@ endif
 ifneq ($(timeout),)
 TEST_FLAGS += -test.timeout=$(timeout)
 else
-TEST_FLAGS += -test.timeout=40m
+TEST_FLAGS += -test.timeout=50m
```

Comment:
Ideally with the PR recently merged to use lighter-weight scrypt params this should go down by quite a bit.

Reply:
Yes, it goes down enough. Dropped this commit.